testsuite/tests/kuadrant/identical_hostnames/conftest.py

```diff
@@ -21,3 +29,9 @@ def route2(request, gateway, blame, hostname, backend, module_label) -> GatewayR
 def authorization_name2(blame):
     """Name of the 2nd Authorization resource"""
     return blame("authz2")
+
+
+@pytest.fixture(scope="module")
+def rate_limit_policy_name2(blame):
```

Comment:
Do you need to reuse the name anywhere? If no I would remove this fixture

Reply:
Not sure if I understand here. This "name2" fixture is used in both test classes and serves as a name of the 2nd RLP that is created via another fixture defined separately in each test class. So I thing this is the proper place for this fixture. Similarly the `authorization_name2` fixture is used for sister "AuthPolicy" tests.

Reply:
Well, what other fixtures (other than `rate_limit_policy2`) use this fixture. If None, then it is not needed and you can generate name in the `rate_limit_policy2` directly.

Reply:
Ah, this is what you meant. I got it know. Make sense, will refactor.

Reply:
Done